### DISCUSSION TITLE
Sonoff Zigbee Bridge to a Router/Repeater

### DISCUSSION THREAD

**Discussion by mboarman** [Koenkk/zigbee2mqtt#12646] [Devices]
I recently  converted my Sonoff Zigbee Bridge to a Router/Repeater using this guide:

https://www.digiblur.com/2022/02/convert-sonoff-zigbee-bridge-to-router-repeater.html

as well as added the necessary custom converter file for Zigbee2mqtt.  When pairing the device everything goes well, however, after a little while it drops off my Zigbee network. I use Zigbee2MQTT and it picks it up as supported and shows as a router in my ZigBee map but after a while it not in the list anymore. I have left the device within a couple feet of the Sonoff Zigbee 3 dongle (my coordinator) to see if it was a distance issue, but it drops every time after an hour or two. Anyone else having this issue?

**Comment by mboarman** [comment]
No one else is having any constant disconnect issues with this firmware and Z2M?

**Reply by AimPizza** [reply]
I do but I also have no idea where to start troubleshooting. Could be due to the firmware or just the nature of Zigbee/wireless in general.
